dashboard/config/routes.rb

```diff
@@ -740,10 +740,15 @@
       get 'peer_review_submissions/index', to: 'peer_review_submissions#index'
       get 'peer_review_submissions/report_csv', to: 'peer_review_submissions#report_csv'
 
-      post 'ml_models/save', to: 'ml_models#save'
-      get 'ml_models/names', to: 'ml_models#user_ml_model_names'
-      get 'ml_models/:model_id', to: 'ml_models#get_trained_model'
-      get 'ml_models/:model_id/metadata', to: 'ml_models#user_ml_model_metadata'
+      resources :ml_models, only: [:show, :destroy] do
```

Comment:
@madelynkasula You suggested I use resource for the ml_models routes on an older PR - just now getting to it.  Is this what you were thinking?

Reply:
yes, this is exactly what i was thinking! thank you for following up on that